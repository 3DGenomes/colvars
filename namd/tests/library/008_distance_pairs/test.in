colvarsTrajFrequency    1
colvarsRestartFrequency 2000

colvar {
    name dmat

    width 0.2

    outputAppliedForce yes
    distancePairs {
        forceNoPBC yes
        group1 {
            psfSegID BH
            atomNameResidueRange CA 1-2
        }
        group2 {
            psfSegID BH
            atomNameResidueRange CA 9-10
        }
    }
} 

harmonic {
    colvars        dmat
    forceConstant  1.0
    centers        (12.0, 14.0, 10.0, 12.0)
    targetCenters  (5.0, 5.0, 5.0, 5.0)
    targetNumSteps 1000
    outputCenters  yes
}
