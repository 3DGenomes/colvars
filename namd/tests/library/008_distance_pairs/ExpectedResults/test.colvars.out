colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013 in any publication based on this calculation.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 2000
colvars: # colvarsTrajAppend = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = dmat
colvars:   Initializing a new "distancePairs" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = on
colvars:     Computing distance using absolute positions (not minimal-image)
colvars:       Initializing atom group "group1".
colvars:       # scalable = off [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group1" defined, 2 atoms initialized: total mass = 24.022, total charge = -0.03.
colvars:       Initializing atom group "group2".
colvars:       # scalable = off [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group2" defined, 2 atoms initialized: total mass = 24.022, total charge = 0.14.
colvars:   All components initialized.
colvars:   # width = 0.2
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = off [default]
colvars:   # outputAppliedForce = on
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { dmat }
colvars:   # outputEnergy = off [default]
colvars:   # forceConstant = 1
colvars:   # centers = { ( 12 , 14 , 10 , 12 ) }
colvars:   # targetCenters = { ( 5 , 5 , 5 , 5 ) }
colvars:   # targetForceConstant = 0 [default]
colvars:   # targetNumSteps = 1000
colvars:   # targetNumStages = 0 [default]
colvars:   # targetForceExponent = 1 [default]
colvars:   # outputCenters = on
colvars:   # outputAccumulatedWork = off [default]
colvars:   The force constant for colvar "dmat" will be rescaled to 25 according to the specified width.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (4 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Re-initialized atom group dmat:0/0. 2 atoms: total mass = 24.022.
colvars: Re-initialized atom group dmat:0/1. 2 atoms: total mass = 24.022.
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars: Updating NAMD interface:
colvars: updating atomic data (4 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Saving collective variables state to "test.colvars.state".
